acceptancetests/jujupy/client.py

```diff
@@ -2121,6 +2122,19 @@ def switch(self, model=None, controller=None):
             raise ValueError('No target to switch to has been given.')
         self.juju('switch', (':'.join(args),), include_e=False)
 
+    def scp(self, _from, _to):
```

Comment:
A couple things:
  - Why do from and to have leading underscores? (they're not 'private' variables etc.)
  - "-m .." is not needed as ModelClient.juju() will add that to the command arguments (unless include_e is False).

I'm not sure there is value adding scp() to ModelClient (seeing as the -m addition is not needed). This could be done in the test as client.juju('scp', (to, from)). It would also need to expose '--no-host-key-checks' and '--proxy' options to be complete.

Reply:
1. `from` can not be used as var name, so we usually added `_` like `_from` or `from_`, so keep consistency, I added `_` for `to` as well.
2. good point, if we are under current client, `model name` is not needed.

Reply:
removed